governance/tag-attribution.md

```diff
@@ -0,0 +1,63 @@
+# TAG Attribution Guidelines
+
+[Attribution] is the act of ascribing a work to a particular author.
+
+The TAG may be attributed with authorship of a work following a robust
+community review and approval process. Depending on the document category, a
+work will require different review processes before attribution is possible.
+
+Below is a list of works categories, and what will enable it to be attributed
+as a TAG publication.
+
+1. Publications (whitepapers, etc)
+  - The document must follow all guidelines, protocols, and processes outlined
+    in the [publications] directory.
+1. Presentations (conference talks, webinars, etc)
+  - The presentation must be made by a TAG Tech Lead or Chair
+  - The presentation content must be pre-approved by a majority of TAG Leads
+  - No significant concern has been left unaddressed by a TAG Tech Lead
+1. Blogs (On the TAG site or elsewhere)
+  - The Blog must be directly connected to an initiative such as a whitepaper
+    or event.
+  - If the blog contains any opinions, advice, or recommendations, it follows
+    all guidelines, protocols, and processes outlined in the [publications]
+    directory.
+
+## Personal Attribution
+
+A TAG Tech Lead, Chair, or project may use their title in any internal or
+external publication as they see fit. There is no restriction to this, but
+the following examples will aid in clarity to avoid mistaken attribution.
+
+### Attribution Qualifier
+
+It is prefferable, but not required, that personally attributed works include
+a qualifying statement if the CNCF or the TAG is mentioned in the author byline
+or article content.
+
+> The author is a leader in CNCF's Technical Advisory Group for Security,
+  but this work is the sole opinion of the author and does not represent
+  any stance from CNCF or the TAG.
+
+### Good Examples
+
+These examples clearly delineate the author attribution from the author's role
+
+- R. Raccoon - Tech Lead, CNCF TAG Security
+- R. Raccoon (Tech Lead, CNCF TAG Security)
```

Comment:
can you add an example with multiple affiliations?

Reply:
Good idea! Applied.